bundles/org.openhab.binding.juicenet/README.md

```diff
@@ -0,0 +1,227 @@
+# JuiceNet Binding
+
+The JuiceNet binding will interface with the cloud portal to get status and manage your JuiceBox EV charger(s).
+In addition to getting the status of various items from the EV charger, it is also possible to start and stop charging sessions.
+
+## Supported Things
+
+This binding supports the following things:
+
+| thing               | type    | description                  |
+|----------           |-------- |------------------------------ | 
+| JuiceNet Account    | Bridge  | This represents the cloud account to interface with the JuiceNet API.  |
+| JuiceBox EV Charger | Device  | This interfaces to a specific JuiceBox EV charger associated with the JuiceNet account. |
+
+This binding should work with multiple JuiceBox EV chargers associated with the account, however it is currently only tested with a single EV charger.
+
+## Thing Configuration
+
+The only configuration required is to create a JuiceNet account thing and fill in the appropriate API token.
+The API token can be found on the Account page at https://home.juice.net/Manage.
+
+Once, the JuiceNet Account thing has been created, all JuiceBox EV chargers associated with that account will be discovered and added to the inbox.
+
+## Manual Configuration Example
+
+The bridge and device can be manually configured in a .things file.
+Both the apiToken parameter for the Bridge and the unitID parameter for the EV charger can be found on the account page at https://home.juce.net.
+
+```
+Bridge juicenet:account:myaccount [ apiToken="xxxx-xxxx-xxxx-xxxx-xxxxx" ] {
+    Thing device JamesCharger [ unitID="xxxxxxx" ]
+}
+```
+
+## Channels
+
+| channel           | type              | read-only | description                  |
```

Comment:
Minor, but consider reformatting alignment of table, so it would also look pretty in source form.